static/elements/chromedash-featurelist.html

```diff
@@ -33,231 +35,280 @@
       case '>=':
         return milestone >= version;
         break;
-      case '=': // Support both '=' and '=='.
+      case '=':  // Support both '=' and '=='.
+      // Support both '=' and '=='.
       case '==':
         return milestone == version;
         break;
       default:
         return false;
+      }
     }
-  }
+    var featureLiList_ = [];
+    Polymer({
+      is: 'chromedash-featurelist',
+
+      properties: {
+        features: {
+          type: Array,
+          notify: true,
+          observer: 'featuresChanged',
+          value: function() { return []; }
+        },
+        filtered: {
+          type: Array,
+          observer: 'filteredChanged',
+          value: function() { return []; }
+        },
+        metadata: {
+          type: Object,
+          value: null,
+          observer: 'metadataChanged'
+        },
+        search: { value: null },
+        whitelisted: {
+          type: Boolean,
+          value: false
+        },
+        // lastStableRelease: {
+        //   type: Number
+        // }
+      },
 
-  var featureLiList_ = [];
+      listeners: {
+        'scroll': 'onScrollList',
+        'keyup': 'onKeyUp',
+        'feature-toggled': 'onFeatureToggled'
+      },
 
-  Polymer({
-    whitelisted: false,
-    metadata: null,
-    search: null,
-    created: function() {
-      this.features = this.features || [];
-      this.filtered = this.filtered || [];
-    },
-    onKeyUp: function(e, detail, sender) {
-      var focusedEl = this.shadowRoot.querySelector(':focus');
-      if (focusedEl) {
-        switch (e.keyCode) {
-          case 13: // enter
+      // observers: [
+      //   '_initialScrollToFeature(features, lastStableRelease)'
+      // ],
+
+      onKeyUp: function (e, detail) {
+        var focusedEl = Polymer.dom(this.root).querySelector(':focus');
+        if (focusedEl) {
```

Comment:
Unless you ever plan on handling additional keys, I'd rewrite this as

    // Check for the enter key
    if (focusedEl && e.keyCode === 13) {
      focusedEl.open = !focusedEl.open
    }